app/controllers/api/v1/books_controller.rb

```diff
@@ -0,0 +1,14 @@
+module Api
+  module V1
+    class BooksController < ApiController
+      include Wor::Paginate
+      def index
+        render_paginated BooksController
```

Comment:
Here isn't BooksController, you are indicating to "render paginated" which model you want to paginate & render